src/Resources/app/administration/src/module/my-parcel/page/sw-myparcel-orders/index.js

```diff
@@ -250,6 +250,19 @@ Component.register('sw-myparcel-orders', {
 
                             gridItem.innerHTML = length.toString();
                         }
+                        if (response.success === false) {
+                            gridItem.innerHTML = '<span class="sw-label sw-label--appearance-pill sw-label--size-default sw-label--warning">Error</span>';
+
+                            gridItem.children[0].onclick = (e) => {
+                                e.preventDefault();
+                                e.stopPropagation();
+                                return false;
+                            }
+
+                            if (response.error) {
+                                gridItem.children[0].innerHTML = response.error;
```

Comment:
hier kan je firstChild voor gebruiken, is iets minder wonky. Maar wat misschien nog beter is dat je het een beetje ombouwt:

```js
const content = response.error ? response.error : 'Error';

gridItem.innerHTML = `<span class="sw-label sw-label--appearance-pill sw-label--size-default sw-label--warning">${content}</span>`;
```